rpcs3/Emu/RSX/Common/surface_store.h

```diff
@@ -543,7 +545,7 @@ namespace rsx
 				(*primary_storage)[address] = std::move(new_surface_storage);
 			}
 
-			verify(HERE), !old_surface_storage, new_surface->get_spp() == get_format_sample_count(antialias);
+			verify(HERE), not_old_surface_storage, new_surface->get_spp() == get_format_sample_count(antialias);
```

Comment:
No, this is not the intended purpose. Leave it as it was.

Reply:
Ok, I'll revert. But it was moved on line 482. May be leave some comment?

Reply:
It's an assert. It's there to make sure things go wrong if I make logic errors during coding so it is redundant by design.

Reply:
In this case, leaving an object in the container is so serious that I felt it necessary to add an assert for it there.

Reply:
Yea, I got it about assert stuff here. But I a little bit haste with my conclusion. If move semantics are implemented for such approach then it's OK. My fault.

BTW what about my second question (cellSpurs)?

Reply:
I cannot confirm about cellSpurs issue. I do not know what is expected for the constants.